### ISSUE TITLE
aws_elasticache_cluster does not support the replication_group_id parameter

### ISSUE THREAD

**Issue by delphyne** [terraform-providers/terraform-provider-aws#2218]
With AWS Provider Plugin version 1.2.0, it is not possible to add replicas to (or remove from) an existing aws_elasticache_replication_group by adding/deleting additional aws_elasticache_cluster resources.  This functionality is available via aws-cli and boto3.

The problem is currently two things:

The following parameters are required in terraform (but not in aws-cli/boto):
- engine
- node_type
- num_cache_nodes
- port

Using the replication_group_id parameter causes error output during the plan phase.

### Terraform Version
v0.10.8

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_elasticache_cluster

### Terraform Configuration Files
```hcl
variable "num_replicas" {
  default = 3
}

resource "aws_elasticache_replication_group" "example_replication_group" {
  replication_group_id = "example"
  replication_group_description = "Managed by Terraform"
  number_cache_clusters = 2
  node_type = "cache.m4.large"
  port = 6379
  apply_immediately = true
  lifecycle {
    ignore_changes = ["number_cache_clusters"]
  }
}

resource "aws_elasticache_cluster" "example_replicas" {
  count = "${var.num_replicas - 1}"
  cluster_id = "${aws_elasticache_replication_group.example_replication_group.id}-r${count.index}"
  replication_group_id = "${aws_elasticache_replication_group.example_replication_group.id}"
  apply_immediately = true
}
```

### Debug Output
N/A

### Panic Output
N/A

### Expected Behavior
Cache clusters should be added or removed as var.num_replicas changes.

### Actual Behavior
> Errors:
>  * aws_elasticache_cluster.example_replicas: "engine": required field is not set
>  * aws_elasticache_cluster.example_replicas: "node_type": required field is not set
>  * aws_elasticache_cluster.example_replicas: "num_cache_nodes": required field is not set
>  * aws_elasticache_cluster.example_replicas: "port": required field is not set
>  * aws_elasticache_cluster.example_replicas: "replication_group_id": this field cannot be set

### Steps to Reproduce
1. `terraform plan`

**Comment by bflad** [comment]
The `aws_elasticache_cluster` support for `replication_group_id` has been merged into master and will be released in v1.13.0 of the AWS provider, likely at the end of next week.

**Comment by bflad** [comment]
This has been released in [version 1.13.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1130-march-28-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
